src/core/lib/gpr/alloc.h

```diff
@@ -25,4 +25,12 @@
 #define GPR_ROUND_UP_TO_ALIGNMENT_SIZE(x) \
   (((x) + GPR_MAX_ALIGNMENT - 1u) & ~(GPR_MAX_ALIGNMENT - 1u))
 
+#define GPR_ROUND_UP_TO_CACHELINE_SIZE(x) \
+  (((x) + GPR_CACHELINE_SIZE - 1u) & ~(GPR_CACHELINE_SIZE - 1u))
+
+#define GPR_ROUND_UP_TO_SPECIFIED_SIZE(x, align) \
+  (((x) + align - 1u) & ~(align - 1u))
```

Comment:
Suggest parens around `align`, in both occurrences.  That way, if someone passes a complex expression as the `align` parameter of the macro, it won't break the expression due to operator precedence rules.

Reply:
Done.